src/redux/themeSlice.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { localStorageKeys } from '../auxFiles/localStorageKeys';
+
+const {
+  themesKeys: {
+    OSTheme,
+    theme,
+  },
+} = localStorageKeys;
+
+const currentOSTheme = localStorage.getItem(OSTheme) || '';
+const themeNameFromLocalStorage = localStorage.getItem(theme) || '';
```

Comment:
move local storage call logic in a component. Slice is not appropriate place to make side effects.